istio.VERSION

```diff
@@ -13,6 +13,6 @@ export AUTH_DEBIAN_URL="https://storage.googleapis.com/istio-artifacts/auth/d10e
 export PILOT_DEBIAN_URL="https://storage.googleapis.com/istio-artifacts/pilot/3101ea9d82a5f83b699c2d3245b371a19fa6bef4/artifacts/debs"
 export PROXY_DEBIAN_URL="https://storage.googleapis.com/istio-artifacts/proxy/2458e4e27bc588df858711b66de0970183e0cf66/artifacts/debs"
 export FORTIO_HUB="docker.io/istio"
-export FORTIO_TAG="0.4.2"
+export FORTIO_TAG="0.6.1"
```

Comment:
Do we still need this, and for what ? 

(this goes for the entire file)

Reply:
it's used in e2e still afaik